README_DEVELOPERS.md

```diff
@@ -68,6 +73,26 @@ drone exec --matrix TARGET=test
 
 ### In a Cloud CI
 
-We provide configuration YAML files for [Travis CI](https://travis-ci.org/) ([.travis.yml](.travis.yml)) and [Shippable](https://app.shippable.com/) ([shippable.yml](shippable.yml)). Enable one of them for your forked repo (we recommend Travis). 
+We provide configuration YAML files for [Travis CI](https://travis-ci.org/) ([.travis.yml](.travis.yml)) and [Shippable](https://app.shippable.com/) ([shippable.yml](shippable.yml)). Enable one of them for your forked repo (we recommend Travis).
 
 Please make sure that each pull request passes all automatic checks before submitting it.
+
+## Commit Message Guidelines
+
+We have a few rules of formatting commit messages and pull request titles:
+
+- Use the imperative mood in the subject line
+- Capitalize the subject line
+- Do not end the subject line with a period
+- Limit the subject line to 50 characters
+- Separate the subject from the body with a blank line
+- Wrap the body at 72 characters
+- Use the body to explain what and why vs how
+- Write the ID of the issue/ticket at the end of the subject line
```

Comment:
Do we encourage brackets or is it enough just to append the ID?

Reply:
I used brackets in the examples. But I think that is not necessary, because of we have a prefix in ID. The main point - approve a single option.

Reply:
A single option would not work:

- "Refactoring after #XXX"
- "Improve performance of whatever (resolves T123456)"
- "Revert stupid changes introduced in #XXX, add whatever"
- "Blablabla (closes #XXX) (#YYY)" - the usual pattern of an issue id + PR id

Reply:
This rule needs just to regulate the ID, that used not in the sentence context. We can clarify it.